.github/workflows/ubuntu-images.yaml

```diff
@@ -10,52 +18,79 @@ on:
   schedule:
     - cron: '0 0 * * MON'
 
-# Prevent multiple workflow runs from racing
-concurrency: ${{ github.workflow }}
-
 env:
-  latest_release: '22.04'
+  distro: 'ubuntu'
+  latest_release: '23.10'
```

Comment:
My preference would be to leave this set to latest LTS :)

Reply:
Good catch. Updated.